libfwupdplugin/fu-context.c

```diff
@@ -703,19 +776,28 @@ fu_context_security_changed(FuContext *self)
  * Since: 1.6.0
```

Comment:
this should be bumped up as we're (essentially) changing ABI for this symbol now.

Reply:
Agree, fixing.